components/logins/examples/sync_pass_sql.rs

```diff
@@ -466,12 +467,15 @@ fn main() -> Result<()> {
             }
             'S' | 's' => {
                 log::info!("Syncing!");
-                if let Err(e) = engine.sync(&client_init, &root_sync_key) {
+                let mut telem_sync = telemetry::Sync::new();
+                if let Err(e) = engine.sync(&client_init, &root_sync_key, &mut telem_sync) {
                     log::warn!("Sync failed! {}", e);
                     log::warn!("BT: {:?}", e.backtrace());
                 } else {
                     log::info!("Sync was successful!");
                 }
+                telem_sync.finished();
+                log::info!("Sync telemetry: {}", serde_json::to_string(&telem_sync).unwrap());
```

Comment:
`serde_json::to_string_pretty` is probably more appropriate here, since we're in a CLI situation.